cf-agent/verify_files_utils.c

```diff
@@ -122,8 +122,11 @@ void SetFileAutoDefineList(const Rlist *auto_define_list)
     AUTO_DEFINE_LIST = auto_define_list;
 }
 
-void VerifyFileLeaf(EvalContext *ctx, char *path, struct stat *sb, Attributes attr, const Promise *pp, PromiseResult *result)
+void VerifyFileLeaf(EvalContext *ctx, char *path, struct stat *sb, const Attributes *attr, const Promise *pp, PromiseResult *result)
 {
+    // FIXME: This function completely ignores it's attr argument
```

Comment:
The FIXME alert is intentional. This function doesn't respect it's attr, argument (it overwrote it), I didn't introduce it, but felt it was important enough to add a FIXME, not just TODO.